R/paste_skip_na.R

```diff
@@ -0,0 +1,64 @@
+#' Concatenate strings dropping missing values
```

Comment:
What do you think about:

 > Like \code{paste()} but missing values are omitted.

Since the docs point mostly to `paste()` already.

Reply:
That's better!